packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -253,6 +261,21 @@ export const Datepicker = React.forwardRef<HTMLDivElement, DatepickerProps>((pro
       : '',
   );
 
+  const trySetOpenState = (
+    newValue: boolean,
+    event:
+      | React.SyntheticEvent
```

Comment:
```

is SyntheticEvent necessary? if so there's no need for the other React events because they just extend that type

Reply:
removing it caused build errors, so keeping for now